### PR TITLE
adds get method for article comments endpoint

### PR DESCRIPTION
**Pull request by digital-cowboy-91**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -104,6 +104,71 @@ describe("/api/articles/:article_id", () => {
...
+      return request(app)
+        .get(base(1))
+        .expect(200)
+        .then((res) => {
+          const { comments } = res.body;
+
+          comments.reduce((prevDate, { created_at }) => {
+            const thisDate = new Date(created_at).getTime();
+
+            expect(prevDate >= thisDate).toBe(true);
```
Comment (nc-nick-jones) [inline __tests__/app.test.js:145]: you can use jest sorted. it's much cleaner!

File: __tests__/app.test.js [hunk 2]
```diff
@@ -104,6 +104,71 @@ describe("/api/articles/:article_id", () => {
...
+
+            return thisDate;
+          }, Infinity);
+        });
+    });
```
Comment (nc-nick-jones) [inline __tests__/app.test.js:150]: just need a test for an article that exists but has no comments.  I see you have actually created this behaviour but just not written a test for it.

**Review by nc-nick-jones — COMMENTED** [review]
see comments for feedback

File: __tests__/app.test.js (hunk 1 above)
Comment (digital-cowboy-91) [reply __tests__/app.test.js:145]: I choose not to introduce unnecessary packages, unless I really need them or it is a stated requirement.

File: __tests__/app.test.js (hunk 2 above)
Comment (digital-cowboy-91) [reply __tests__/app.test.js:150]: FIX PR: https://github.com/digital-cowboy-91/nc-backend-project/pull/10

**Pull request merged**